guides/leave_space_mutation.md

```diff
@@ -0,0 +1,49 @@
+---
+copyright: 'Copyright IBM Corp. 2017'
+link: 'leave-space-mutation'
+is: 'experimental'
+---
+
+# Leave Space
+
+## Concepts
+
+Currently this API only supports invited space members, for other members it will return an error message. The _leaveSpace_ mutation allows you to leave a workspace. The mutation accepts a Space ID as an argument and makes the request on behalf of the calling user.
```

Comment:
How about:
"The _leaveSpace_ mutation allows the user to leave a space. The mutation accepts a Space ID as an argument and makes the request on behalf of the calling user. Currently this API can only be used to reject an invitation to a space. Otherwise it will return an error."